### DISCUSSION TITLE
Column filters rendered outside of column?

### DISCUSSION THREAD

**Discussion by e-monson** [TanStack/table#3366] [Q&A]
I need a search input that displays outside of my table, but it will only filter using data from one column. Is it possible to render a column filter outside of its column?

**Comment by bruno-fialho** [comment]
I need something like this. In my case, I have a table of products, with a column with two possible values "active" or "inactive", and I need a button on top of the table to show/hide inactive products rows.

**Comment by bruno-fialho** [comment]
@e-monson I just found a couple of solutions: https://github.com/tannerlinsley/react-table/discussions/2181

**Comment by buglessir** [comment]
@bruno-fialho , @e-monson  check this page:
https://react-table.tanstack.com/docs/api/useFilters
you can handle this feature with `setFilter: Function(columnId, filterValue) => void`
extract `setFilter` via `useTable` and set a new value outside of the internal filters under headers just with a simple `onChange` action.
I've tried and is working for me !
